### PR TITLE
Add option to return jira issue metadata (Issue #94)

### PR DESCRIPTION
**Pull request by gs202**
Adding option to return jira issue model (with metadata) instead of boolean result (`--jira-return-issue`) issue #94
The issue type that will be returned is `JiraIssue` model using pydantic, with currently the MVP fields.

Using jira (https://pypi.org/project/jira/) will require changing the connection method, also the returned fields (metadata) has nothing more than when using request call - so i thought it will not add any value.

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+from typing import List, Union
```
Comment (lukas-bednar) [inline models.py]: Name of this file is too generic, what about something like `jiramodels` or something like that.

**Comment by gs202** [comment]
> Thank you for your contribution, it looks like that `pydantic` lib is not really available for py3.5.
> 
> I don't like idea of loosing compatibility with supported version of python.
> You might consider to restrict this feature to be enabled only for `>py3.5` ... or something like that.

Will work on that

File: README.rst [hunk 2]
```diff
@@ -153,6 +153,7 @@ Requires
 -  requests >= 2.13.0
 -  six
 -  retry>=0.9.2
+-  pydantic>=0.32.2
```
Comment (liiight) [inline README.rst]: you can use dataclasses module, it has a backport for python 3.5

Comment (gs202) [reply README.rst]: Sounds good, i'll start a new PR with it. Thanks!

**Pull request closed without merging**